src/libraries/System.Runtime/tests/System/UIntPtrTests.cs

```diff
@@ -547,6 +548,49 @@ public static void Parse_Span_Invalid(string value, NumberStyles style, IFormatP
             }
         }
 
+        [Theory]
+        [MemberData(nameof(Parse_ValidWithOffsetCount_TestData))]
+        public static void Parse_Utf8Span_Valid(string value, int offset, int count, NumberStyles style, IFormatProvider provider, nuint expected)
+        {
+            nuint result;
+            ReadOnlySpan<byte> valueUtf8 = Encoding.UTF8.GetBytes(value, offset, count);
+
+            // Default style and provider
+            if (style == NumberStyles.Integer && provider == null)
+            {
+                Assert.True(nuint.TryParse(valueUtf8, out result));
+                Assert.Equal(expected, result);
+            }
+
+            Assert.Equal(expected, nuint.Parse(valueUtf8, style, provider));
+
+            Assert.True(nuint.TryParse(valueUtf8, style, provider, out result));
+            Assert.Equal(expected, result);
+        }
+
+        [Theory]
+        [MemberData(nameof(Parse_Invalid_TestData))]
+        public static void Parse_Utf8Span_Invalid(string value, NumberStyles style, IFormatProvider provider, Type exceptionType)
+        {
+            if (value != null)
+            {
+                nuint result;
+                ReadOnlySpan<byte> valueUtf8 = Encoding.UTF8.GetBytes(value);
+
+                // Default style and provider
+                if (style == NumberStyles.Integer && provider == null)
+                {
+                    Assert.False(nuint.TryParse(valueUtf8, out result));
+                    Assert.Equal(default, result);
+                }
+
+                Assert.Throws(exceptionType, () => nuint.Parse(Encoding.UTF8.GetBytes(value), style, provider));
+
+                Assert.False(nuint.TryParse(valueUtf8, style, provider, out result));
+                Assert.Equal(default, result);
+            }
+        }
```

Comment:
Code coverage of corelib should be working again.  If you haven't already, could you just double-check that these tests are sufficient? In particular, there are all of those numbers factored out of float/double/Half to dedup the floating-point parsing, and it'd be easy for a mistake to accidentally slip in there.  It'd be good to know they're all at least minimally exercised.